src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Account/Pages/ResetPassword.razor

```diff
@@ -72,15 +70,13 @@
         if (user is null)
         {
             // Don't reveal that the user does not exist
-            RedirectManager.RedirectTo("/Account/ResetPasswordConfirmation");
-            return;
+            RedirectManager.RedirectTo("Account/ResetPasswordConfirmation");
         }
 
         var result = await UserManager.ResetPasswordAsync(user, Input.Code, Input.Password);
         if (result.Succeeded)
         {
-            RedirectManager.RedirectTo("/Account/ResetPasswordConfirmation");
-            return;
```

Comment:
It's kind of subtle that `RedirectTo()` never returns, making the `return` statement redundant. From reading the code, it might be hard to tell at first that each of these `if` statements acts as a kind of implicit guard clause. But I don't feel a strong way about this so if you think it's fine to remove the `return`s then I won't fight it! 🙂